CHANGELOG.rst

```diff
@@ -1,5 +1,36 @@
 .. _changelog:
 
+0.22.0
+------
+
+Renku ``0.22.0`` updates the core-service and CLI components to version 2.0, adding support for workflow files
+and requiring users to migrate their project metadata through the UI to version 10.
+It also phases out support for projects on metadata version 8, meaning those projects must be migrated to have
+full functionality in the UI.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🎨 **Renku CLI**: Add support for writing workflow files which can be executed directly, greatly improving the UX
+  around workflows
+  (`#3176 <https://github.com/SwissDataScienceCenter/renku-python/pull/3176>`_)
```

Comment:
```suggestion
**🌟 New Features**

* 📝 **Renku CLI**: Introduce the Renku Workflow File, a yaml-based file for defining Renku workflows (`docs <https://renku.readthedocs.io/en/workflow-file-documentation/topic-guides/workflows/workflow-file.html>`_)
  (`#3176 <https://github.com/SwissDataScienceCenter/renku-python/pull/3176>`_)

**✨ Improvements**

```

Reply:
same comment as above - probably safer to use `latest`